midi/midi/decode.py

```diff
@@ -350,17 +362,18 @@ def export_output(folder: str,
     np.save('{}/{}'.format(folder, filename), output)
 
 
-def combine_and_clean_tracks(tracks: list[MidiTrack]):
+def combine_and_clean_tracks(tracks: list[MidiTrack]) -> MidiTrack:
     """
     translates all Messages to a single list, then removes vacuous Messages
 
     :param tracks:
     :return:
     """
     note_grid = [0] * 128                            # list with numbers of active notes, one for each height
-    raw_messages = list[tuple[int, Message]]()       # all messages with their starting times
-    filtered_messages = list[tuple[int, Message]]()  # as raw_messages, but without repetitions
+    raw_messages = list[Tuple[int, Message]]()       # all messages with their starting times
+    filtered_messages = list[Tuple[int, Message]]()  # as raw_messages, but without repetitions
```

Comment:
Why this change?

Reply:
it turns out a component (don't remember which one) fails at complex tuples to correctly count the number of arguments in a tuple (CI threw a few errors)